### DISCUSSION TITLE
Flagged Account: A failure story

### DISCUSSION THREAD

**Discussion by killerjoker73** [community/community#142305] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

# Abstract

It’s been over a month since my GitHub account was flagged. I’m @satyam-mishra-pce, and I’m sharing this from my friend's account because flagged accounts can't participate in public discussions (they can, but the posts are not visible publicly). 
This is my journey, where I’ve gone further than I ever thought possible to get my account unflagged, only to find dead ends at every turn.

# The first attempts: A dead end

It started more than a month ago when my GitHub account was suddenly flagged for reasons still unknown to me. I could access it, but it wasn’t publicly visible. My friend, @codehamsters, had faced the same issue before, so I reached out to him for advice. Unfortunately, his story wasn’t reassuring. He told me how he’d tried everything, but nothing worked:

1.  Submitted a support ticket — two months, and no response.
2.  Tried transferring ownership of repositories — not available for flagged accounts.
3.  Looked for ways to move data from the flagged account to another — also not available.
4.  No other support channels seemed open.

I panicked. I had so many repositories, and I was in the middle of applying for jobs, with my GitHub profile on my resume. Yet, I thought I’d still give it my best shot. I opened a support ticket (now a month old) and waited… and waited, but no response.

# Exploring ways

That’s when I had an idea. What if I upgraded to GitHub Pro? Surely that would get me the support I needed. But no — flagged accounts can’t upgrade. I didn’t give up. I asked my organization partner to sign up for GitHub Teams. It was $4 a month, and I was ready to try anything. We opened a ticket, mentioning that one of our team members had a flagged account. We were hopeful. But hours turned into days, and still, nothing.

# Out of the box

While waiting for any support response, I decided to think outside the box. I saw that GitHub Enterprise Plan worth $21 (with an option of one month trial) offered a 24-hour reply window, and there was an option to contact the team for help choosing the right plan. I thought, why not use this as a way to reach them? So, I used my university email, hoping it would add some credibility and make them more likely to respond. It felt like a creative, out-of-the-box move — one that just might work. But despite this effort, a full week passed, and I still didn’t hear back from the so-called "Premium Support Team."

# Enterprise: The last hope?

Despite the doubts, I decided to try the Enterprise trial. I opened a support ticket, but the system automatically tagged it as “Low Priority.” After some digging, I found that only major issues like server outages or organizational lockouts get urgent priority. Still, I managed to bump it to “Normal Priority” and waited again.

# The silence saga

Meanwhile, the Premium Support response finally arrived. But all they did was ask questions about my use case to recommend a plan. I explained my situation, stressing the lack of support on free and paid plans. It’s been two weeks since I last heard from them.

Back to the Enterprise ticket — I expected a reply in a few days, but after a full week, my ticket wasn’t answered. Instead, it was archived. I felt like I had been stabbed.

# Hopeless

Today, I created a new account, hoping to import my repositories and keep the commit history intact. Forking wasn’t an option, nor was adding collaborators. I tried the import tool and, again, it failed because my original account was flagged.

As I am writing this post, my organization partner's account got flagged throughout the entire process of him helping me recover my account. Shoutout to him (@codehamster69). Thanks mate.

A few minutes ago, I found this GitHub Discussion post, where people were tagging @github-staff to escalate their reinstatement requests. So, here I am, giving it a shot. My ticket is #3010454, and my username is @satyam-mishra-pce.

But honestly, at this point, I’ve lost hope. It seems like this flag is forever, and all my hard work over the years has been for nothing. It’s frustrating and disheartening to feel so powerless, especially when such a huge platform like GitHub, backed by Microsoft, can’t offer any help.

This is my story. A story of failure — mine, and GitHub’s.

_- Satyam Mishra_
![Literally me](https://i.imgur.com/9G9Rdce.png)

**Comment by KhushalJangid** [comment]
That's very unfortunate satyam and @github-staff

**Comment by jiuyueshenhua** [comment]
me tool，it is so bad！

**Comment by ghost** [comment]
Same here my account is @RomThpt and my ticket is #3151909, @github-staff please 🙏

**Comment by ShiiiivanshSingh** [comment]
same
